plugin/src/main/java/io/jenkins/plugins/analysis/warnings/DifferentCompatibilityScanner.java

```diff
@@ -0,0 +1,170 @@
+import io.jenkins.plugins.datatables.TableColumn;
+import io.jenkins.plugins.util.JenkinsFacade;
+
+public class DifferentCompatibilityScanner extends AnalysisModelParser {
+    private static final long serialVersionUID = 5105550534032476746L;
+
+    public static class CompatibilityLabelProvider extends StaticAnalysisLabelProvider {
+
+        public CompatibilityLabelProvider(final String id, final String name) {
+            super(id, name);
+        }
+
+        @Override
+        public DetailsTableModel getIssuesModel(final Run<?, ?> build, final String url,
+                final Report report) {
+            return new CompatibilityModel(report, getFileNameRenderer(build), getAgeBuilder(build, url), this,
+                    new JenkinsFacade());
+        }
+    }
+
+    public static class CompatibilityModel extends DetailsTableModel {
+
+        /**
+         * Creates a new instance of {@link DetailsTableModel}.
+         *
+         * @param report
+         *         the report to render
+         * @param fileNameRenderer
+         *         renders the file name column
+         * @param ageBuilder
+         *         renders the age column
+         * @param descriptionProvider
+         *         renders the description text
+         * @param jenkinsFacade
+         *         Jenkins facade to replaced with a stub during unit tests
+         */
+        protected CompatibilityModel(final Report report,
+                final FileNameRenderer fileNameRenderer,
+                final AgeBuilder ageBuilder,
+                final DescriptionProvider descriptionProvider,
+                final JenkinsFacade jenkinsFacade) {
+            super(report, fileNameRenderer, ageBuilder, descriptionProvider, jenkinsFacade);
+        }
+
+        @Override
+        protected TableRow getRow(final Issue issue) {
+            CompatibilityRow row = new CompatibilityRow(getAgeBuilder(), getFileNameRenderer(), getDescriptionProvider(), issue, getJenkinsFacade());
+            final Serializable additionalInfo = issue.getAdditionalProperties();
+            if (additionalInfo instanceof RevApiInfoExtension) {
+                final RevApiInfoExtension revApiInfo = (RevApiInfoExtension) additionalInfo;
+                row.setOldFile(revApiInfo.getOldFile());
+                row.setNewFile(revApiInfo.getNewFile());
+                row.setIssueName(revApiInfo.getIssueName());
+                row.setSeverities(revApiInfo.getSeverities());
+                row.setCategory(issue.getCategory());
+            }
+            row.setSeverity(issue);
+            return row;
+        }
+
+        @Override
+        public String getId() {
+            return "issues";
+        }
+
+        @Override
+        public List<TableColumn> getColumns() {
+            List<TableColumn> columns = new ArrayList<>();
+            columns.add(createDetailsColumn());
+            columns.add(new TableColumn("Name", "issueName").setWidth(2));
```

Comment:
I think you need to rebase your PRs, it seems you are using old and deprecated API. Example of new column API: https://github.com/jenkinsci/warnings-ng-plugin/blob/4be14e0c5840b63ed0e39d624a57b10779314450/plugin/src/main/java/io/jenkins/plugins/analysis/core/model/DetailsTableModel.java#L115